src/skeleton/skeleton.stories.ts

```diff
@@ -0,0 +1,30 @@
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs, number } from "@storybook/addon-knobs/angular";
+
+import { SkeletonModule } from "./skeleton.module";
+
+storiesOf("Components|Skeleton", module).addDecorator(
+	moduleMetadata({
+		imports: [ SkeletonModule ]
+	})
+)
+	.addDecorator(withKnobs)
+	.add("Skeleton Placeholder", () => ({
+		template: `<ibm-skeleton-placeholder></ibm-skeleton-placeholder>`
+	}))
+	.add("Skeleton Text", () => ({
+		template: `
+			<ibm-skeleton-text
+				[lines]="lines"
+				[minLineWidth]="minLineWidth"
+				[maxLineWidth]="maxLineWidth"
+				>
```

Comment:
We don't normally keep these lonely, we keep them on the previous line :)

Reply:
no longer lonely 😃